### PR TITLE
Add freebie tag and tool tip for forward solve, backsolve, and freebie

### PR DESCRIPTION
**Pull request by maximized**

### CODE REVIEW

**Comment by maximized** [comment]
Most of the changes to the html file were caused by me running 
```
prettier hunts/templates/stats.html --write --config .prettierrc 
```

File: hunts/models.py [hunk 1]
```diff
@@ -47,6 +47,14 @@ def get_object_or_404(user=None, **kwargs):
     def get_num_solved(self):
         return self.puzzles.filter(status=Puzzle.SOLVED).count()
 
+    def get_num_backsolved(self):
+        all_solved = self.puzzles.filter(status=Puzzle.SOLVED)
+        return sum(1 for puzzle in all_solved if puzzle.is_backsolved())
```
Comment (rgossiaux) [inline hunts/models.py:52]: I believe this is going to do a fresh db query per puzzle in order to compute is_backsolved. I think to do one combined query you want to do something like `self.puzzles.filter(status=Puzzle.SOLVED, tags__name=PuzzleTag.BACKSOLVED).count()` (I haven't tested this though). ditto below

**Review by rgossiaux — CHANGES_REQUESTED** [review]
Nice, thanks!

Just a perf comment (I haven't tested your code so I might be wrong, though)

File: hunts/models.py (hunk 1 above)
Comment (maximized) [reply hunts/models.py:52]: This worked

**Review by maximized — COMMENTED** [review]
Addressed comments

**Review by rgossiaux — APPROVED** [review]
Ty, lgtm!

**Pull request merged**